addons/rating/views/rating_templates.xml

```diff
@@ -16,6 +16,16 @@
             </t>
         </template>
 
+        <!-- External page: Invalid partner rating -->
+        <template id="invalid_partner_rating_external_page_view" primary="True"
+            inherit_id="rating.rating_external_page_view" name="Not allows to rating">
+                <xpath expr="//div/h3" position="replace">
+                    <div class="text-center fs-4">
+                        You are not allowed to give the rating for the <t t-out="name"/>
```

Comment:
Maybe re-word to "Only the customer can rate: {name}" ?

The PO will probably have a better suggestion anyway.